Codeinwp/neve
# fix: placement of the loop entry before after hook

### Summary
It Fixes the placement of the loop entry before after the hook.

### Will affect the visual aspect of the product
NO


### Test instructions
- Enable masonry on the blog
- Add a new custom layout to before/after post
- See how it looks on different layouts.


Closes Codeinwp/neve-pro-addon#1918.
